frontend/src/store/reportChanges.js

```diff
@@ -1,14 +1,13 @@
 import { isEmpty } from 'lodash';
```

Comment:
Can we kill the non error related console logs in this file?

Reply:
it's updated